docs/api_call.md

```diff
@@ -1,1352 +1,1108 @@
+    >>> detector.set_min_max_digits(min_digit=min_number_digits,
+                                    max_digit=max_number_digits) 
+    >>> output = detector.detect(message=message,structured_value=structured_value,
+                                fallback_value=fallback_value,
+                                bot_message=bot_message)
+    >> print(output)
+    ```
+
+  - *CURL command:*
+
+    ```shell
+    URL='localhost'
+    PORT=8081
+    
+    curl -i 'http://'$URL':'$PORT'/v2/number/?message=I%20want%20to%20purchase%2030%20units%20of%20mobile%20and%2040%20units%20of%20Television&entity_name=number_of_unit&structured_value=&fallback_value=&bot_message=&min_number_digits=1&max_number_digits=2&source_language=en&unit_type='
+    ```
+
+    > **Output:**
+
+    ```json
+    {"data": [
+        {
             "detection": "message",
-            "original_text": "40",
-            "entity_value": {
-              "value": "40"
-            }
-          }
-        ]
-      }
-      ```
-
-  - Example 2:
-
-    - ```python
-      message = "I want to reserve a table for 3 people"
-      entity_name = 'number_of_people'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      min_digit = 1
-      max_digit = 2
-      ```
-
-    - *Python:*
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_number
-      output = get_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message, min_digit=min_digit, max_digit=max_digit)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/number/?message=I%20want%20to%20reserve%20a%20table%20for%203%20people&entity_name=number_of_people&structured_value=&fallback_value=&bot_message=&min_number_digits=1&max_number_digits=2'
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
+            "original_text": "30", 
+            "entity_value": { "value": "30", "unit": null},
+            "language": "en"
+        },
+        {
+          "detection": "message",
+          "original_text": "40",
+          "entity_value": { "value": "40", "unit": null},
+          "language": "en"
+        }]
+    }
+    ```
+
+- ***Example 2: Detecting number[Hindi] without unit in message***
+
+  - *Django Shell*
+
+    ```python
+    # For a sample query with following parameters
+    >>> message="मुझे ३० किलो आटा और दो हजार का चीनी देना "
+    >>> entity_name='number'
+    >>> structured_value=None
+    >>> fallback_value=None
+    >>> bot_message=None
+    >>> min_number_digits=1  # minimum number of digit 
+    >>> max_number_digits=6  # maximum number of digit
+    >>> source_language='hi' # here language will be ISO 639-1 code
+    >>> unit_type=None       # this restrict the number detector to detect particular 
```

Comment:
Out of all examples, only one example has unit type non null. Also how do I detect two entities, weight and currency. This example is a good one to demonstrate it. Please specify unit.